src/components/Signup Box/signupBox.js

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+import { useState } from "react";
+import { useAuth } from "../../Auth/authContetxt";
+import './signupBox.css'
+import { baseurl } from "../../utils/apiCalls";
+
+export const SignUpBox = () => {
+  const { loginHandler } = useAuth();
+
+  const [fullName, setFullName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  
+  const signUpHandler = async (fullName, userName, email, password, loginHandler) => {
```

Comment:
- This function is taking more than 3 parameters it's a good practice to convert it into an object.
- We should not write `handler` in function name when the name is self explanatory
e.g. signUpHandler -- signUpUser
```suggestion
  const signUpUser = async ({ fullName, userName, email, password, loginHandler }) => {
```